lib/runtime/wasmer/imports.go

```diff
@@ -1742,6 +1743,32 @@ func ext_offchain_http_request_start_version_1(context unsafe.Pointer, methodSpa
 	return C.int64_t(ptr)
 }
 
+//export ext_offchain_http_request_add_header_version_1
+func ext_offchain_http_request_add_header_version_1(context unsafe.Pointer, reqID C.int32_t, keySpan, valueSpan C.int64_t) C.int64_t {
+	logger.Debug("executing...")
```

Comment:
Actually since we log out the function caller, is this actually needed? Maybe we should change the convention :thinking: Let me know I'll create an issue for that.

Reply:
The `logger` shows the caller currently, we don't need the function caller anymore, @qdm12 would be nice to have that issue! could you assign me to the issue?

Reply:
Assigned you to https://github.com/ChainSafe/gossamer/issues/2052

Let's check with the team at standup just to be safe.